### DISCUSSION TITLE
Add the ability to fetch the list of contributors

### DISCUSSION THREAD

**Discussion by adri326** [thislooksfun/snoots#48] [Q&A] [answered]
While I was trying out this library, I hit a roadblock: there seems to be no API for the `/r/<subreddit>/about/contributors` endpoint. I also couldn't find a straightforward way to bridge that gap myself, without going ahead and changing the source code of snoots myself.

Hopefully this can be added in the future!

**Comment by thislooksfun** [comment, marked as answer]
This is one of the many endpoints I have on my list but haven't yet had the time to implement. I'll put this one at the top.

In future, please use the feature request form (one of the options when creating a [new issue](https://github.com/thislooksfun/snoots/issues/new/choose)) so I can track it more easily. I've transferred this to #49, and will track progress there.
